package/snapcraft.yaml

```diff
@@ -0,0 +1,33 @@
+name: konstructs-client
+version: 0
+summary: A voxel based game client
+description: An open source Infiniminer/Minecraft inspired multiplayer game.
+confinement: strict
+grade: devel
+
+apps:
+  konstructs-client:
+    command: konstructs
+    plugs:
+      - x11
+      - network
+      - opengl
+      - unity7
+
+parts:
+  client:
+    plugin: cmake
+    source: ..
+    configflags: [-DCMAKE_INSTALL_PREFIX=/]
+    build-packages:
```

Comment:
can you clarify "The ones that should be used are the ones that are included in our repository"?